proposals/description/swap_dpi_to_dai.ts

```diff
@@ -0,0 +1,43 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const swap_dpi_to_dai: ProposalDescription = {
+  title: 'Swap DPI to DAI',
+  commands: [
+    {
+      target: 'dpi',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{dpiToDaiSwapper}', '37888449801955370645659'],
+      description: 'Transfer DPI from DAO timelock to the LBP pool'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{dpiToDaiSwapper}', '187947000000000000000000'],
+      description: 'Withdraw Use the PCVGuardian to transfer DAI from the CompoundPCVDeposit to the LBP pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDai}'],
+      description: 'Add DAI lens on swapper to CR oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDpi}'],
+      description: 'Add DPI lens on swapper to CR oracle'
+    }
```

Comment:
you can use addDeposit**s** if you have multiple deposits

also you can remove `dpiDepositWrapper` from CR oracle, no need to read DPI held on the DAO Timelock anymore.